oneshot.d/browser-move-data-to-new-location

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# Moves Browser and Captive portal profiles to new location
+OLD_BROWSER_PROFILE_PATH="$HOME/.mozilla/mozembed"
+NEW_BROWSER_PROFILE_PATH="$HOME/.cache/org.sailfishos/sailfish-browser/.mozilla"
+
+OLD_CAPTIVE_PORTAL_PROFILE_PATH="$HOME/.mozilla/captiveportal"
+NEW_CAPTIVE_PORTAL_PROFILE_PATH="$HOME/.cache/org.sailfishos/sailfish-browser-captiveportal/.mozilla"
+
+if [ -e "$OLD_BROWSER_PROFILE_PATH/" ] && [ ! -e "$NEW_BROWSER_PROFILE_PATH/" ]
+then
+    mkdir -p "$NEW_BROWSER_PROFILE_PATH"
+    if mv "$OLD_BROWSER_PROFILE_PATH/"* "$NEW_BROWSER_PROFILE_PATH/"
+    rm -f $OLD_BROWSER_PROFILE_PATH/.parentlock
```

Comment:
This looks like a mistakenly placed line. Similar on the next if then statement.

Reply:
Oops. Fixed.